src/components/BookForm.js

```diff
@@ -0,0 +1,60 @@
+/* eslint-disable no-unused-vars */
```

Comment:
- [ ]  @Stephen-Adom its not good practice disabling linter check in your file, kindly remove this disabler and fix the linter errors